packages/mds-policy-service/@types/models.ts

```diff
@@ -100,6 +101,13 @@ export interface TNCRule<RuleType extends RULE_TYPE = RULE_TYPE>
   modality: 'tnc'
 }
 
+export type MicroMobilityRuleType =
+  | MicroMobilityRule<'count'>
+  | MicroMobilityRule<'time'>
+  | MicroMobilityRule<'speed'>
+  | MicroMobilityRule<'user'>
+  | MicroMobilityRule<'rate'>
```

Comment:
Is this type necessary? I only see it used in tests, and it _should_ be the same as just `MicroMobilityRule` without generic args

Reply:
good cleanup, thanks!